crates/rspack_plugin_javascript/tests/fixtures/provide/snapshot/output.snap

```diff
@@ -47,21 +47,29 @@ function defaultClearTimeout() {
 }
 (function() {
     try {
-        if (typeof setTimeout === "function") cachedSetTimeout = setTimeout;
-        else cachedSetTimeout = defaultSetTimout;
+        if (typeof setTimeout === "function") {
```

Comment:
Snapshot changes because of we don't want to apply `expr_simplifier` and `dead_branch_remover` to every module. This will add additional overhead. Removing this will not change semantic behavior. It works more well with `DefinePlugin`.